.github/workflows/release.yml

```
@@ -4,17 +4,17 @@ on:
     - master
     paths:
+    - .github/workflows/release.yml
     - '**/*.py'
     - '**/*.sh'
-    - Dockerfile
-    - .pre-commit-hooks.yaml
-    # Ignore paths
-    - '!tests/**'
+    - Dockerfile*
+    - .pre-commit-hooks.yaml 
+    paths-ignore:  
```

actionlint (1.7.7)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)